s_tui.py

```diff
@@ -432,63 +480,97 @@ def graph_stats(self):
+                self.controller.script_loader.load_script(
+                    temp_source.__class__.__name__, 30000)
+            )  # Invoke threshold script every 30s while threshold is exceeded.
 
-        if script_hooks_enabled:
-            temp_source.add_edge_hook(self.script_loader.load_script(temp_source.__class__.__name__, 30000)) # Invoke threshold script every 30s while threshold is exceeded.
+        alert_colors = ['high temp light',
+                        'high temp dark',
+                        'high temp light smooth',
+                        'high temp dark smooth']
 
-        alert_colors = ['high temp light', 'high temp dark', 'high temp light smooth', 'high temp dark smooth']
-        self.graphs[temp_source.get_source_name()] = StuiBarGraph(temp_source, 'temp light', 'temp dark', 'temp light smooth', 'temp dark smooth', alert_colors=alert_colors)
-        self.summaries[temp_source.get_source_name()] = SummaryTextList(temp_source, 'high temp txt')
+        self.graphs[temp_source.get_source_name()] = StuiBarGraph(
+            temp_source, 'temp light', 'temp dark',
+            'temp light smooth', 'temp dark smooth',
+            alert_colors=alert_colors
+        )
+
+        self.summaries[temp_source.get_source_name()] = SummaryTextList(
+            temp_source, 'high temp txt')
 
         rapl_power_source = RaplPowerSource()
-        self.graphs[rapl_power_source.get_source_name()] = StuiBarGraph(rapl_power_source, 'power dark', 'power light', 'power dark smooth', 'power light smooth')
-        self.summaries[rapl_power_source.get_source_name()] = SummaryTextList(rapl_power_source)
+
+        self.graphs[rapl_power_source.get_source_name()] = StuiBarGraph(
+            rapl_power_source, 'power dark', 'power light',
+            'power dark smooth', 'power light smooth')
+
+        self.summaries[rapl_power_source.get_source_name()] = SummaryTextList(
+            rapl_power_source)
 
         fan_source = FanSource(self.custom_fan)
-        self.summaries[fan_source.get_source_name()] = SummaryTextList(fan_source)
+        self.summaries[fan_source.get_source_name()] = SummaryTextList(
+            fan_source)
 
         # only interested in available graph
-        self.available_graphs = OrderedDict((key, val) for key, val in self.graphs.items() if val.get_is_available())
-        self.available_summaries = OrderedDict((key, val) for key, val in self.summaries.items() if val.get_is_available())
+        self.available_graphs = OrderedDict(
+            (key, val) for key, val in self.graphs.items()
+            if val.get_is_available())
+        self.available_summaries = OrderedDict(
+            (key, val) for key, val in self.summaries.items() if
+            val.get_is_available())
 
         self.visible_graphs = self.available_graphs.copy()
+        logging.debug("All availabe graphs: " + str(self.visible_graphs))
+
+        # Remove graphs from shown graphs if user configed them out
```

Comment:
There is a duplication here, we read the config again. Could make problems in the future